schema/data/attributes_schema.cue

```diff
@@ -0,0 +1,18 @@
+base_attributes!: [
+	...{
+		id!:          int
+		name!:        string
+		friendly_id!: string
+		handle!:      string
+		values!: [_, ...string] // at least one; all strings
```

Comment:
Yeah that could be a good add ✅ 

For the most part, keeping this focused on schema validation (since ruby tests ensure validity). But a regex check is easy